### PR TITLE
Update stencil to latest build

### PR DESCRIPTION
**Pull request by caseydierking**
Manually updated Stencil to 0.13.2 and removed Jest and @type/jest dependencies. Also removed configuration for Jest.

### CODE REVIEW

**Comment by Dohxis** [comment]
I saw your last PR, you can keep updating the same one by just pushing more things without creating new ones. I will close the previous one.

File: package.json [hunk 1]
```diff
@@ -33,20 +33,14 @@
...
 		"type": "git",
 		"url": "git+https://github.com/Dohxis/air-datepicker.git"
 	},
 	"homepage": "https://github.com/Dohxis/air-datepicker",
-	"jest": {
-		"transform": {
-			"^.+\\.(ts|tsx)$": "<rootDir>/node_modules/@stencil/core/testing/jest.preprocessor.js"
-		},
-		"testRegex": "(/__tests__/.*|\\.(test|spec))\\.(tsx?|jsx?)$",
+	"testRegex": "(/__tests__/.*|\\.(test|spec))\\.(tsx?|jsx?)$",
```
Comment (Dohxis) [inline package.json]: testRegex is part of jest configuration which can be removed.

**Comment by Dohxis** [comment]
Also you removed changes you made for tests scripts which were correct. Please put those back.

**Comment by caseydierking** [comment]
Thanks for bearing with me here. Still learning how to pull and contribute thanks to hacktoberfest. I made those changes and pushed!

**Comment by caseydierking** [comment]
Anything more I need to do here?

**Pull request closed without merging**